recipe/meta.yaml

```diff
@@ -5,28 +5,30 @@ package:
   version: {{ version }}
 
 source:
-  url: https://github.com/neo4j/neo4j-python-driver/archive/{{ version }}.tar.gz
-  sha256: 60e5a529e1262a1d73ff378c337d03dcf7bbd84b638e0ce5ace2f90846cdddc7
+  url: https://pypi.io/packages/source/n/neo4j/neo4j-{{ version }}.tar.gz
+  sha256: b6c49fbd60426e268ed4afbd414766444fe70aee1ac0376a0c871d75526b8251
 
 build:
-  number: 0
+  number: 1
   noarch: python
   script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
-  build:
-    - {{ compiler('c') }}
   host:
-    - python
-    - cython
```

Comment:
As I commented in the PR description, I don't think you want to keep the C compiler and `cython` in a `noarch: python` package.  The Linux compiled code wouldn't work on OSX or Windows.